devicemodel/core/main.c

```diff
@@ -854,6 +855,10 @@ main(int argc, char *argv[])
 	while ((c = getopt_long(argc, argv, optstr, long_options,
 			&option_idx)) != -1) {
 		switch (c) {
+		case 'A':
+			pr_info("The '-A' parameter is obsolete and ignored. "
```

Comment:
The construction looks odd, how does it get printed on the terminal? Can you paste a screenshot of that?

Reply:
The '-A' parameter is obsolete and ignored. Virtual ACPI tables are always enabled.
logger: name=console, level=4
logger: name=kmsg, level=3